lib/site_encrypt/phoenix.ex

```diff
@@ -36,7 +36,7 @@ defmodule SiteEncrypt.Phoenix do
   end
 
   defp acme_server_adapter_spec(Plug.Adapters.Cowboy, port),
-    do: {Plug.Adapters.Cowboy, scheme: :http, options: [port: port, acceptors: 1]}
+    do: {Plug.Adapters.Cowboy, scheme: :http, options: [port: port, transport_options: [num_acceptors: 1]]}
```

Comment:
This won't work with Plug 1.5 though, so I think we should keep using the `acceptors` option. The fact that it's deprecated doesn't mean it won't work, and the warning will be issued only in your application on the first compilation.